src/pages/Books.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Books() {
+  return (
+    <div>
+      <h1>Books Page Component</h1>
```

Comment:
- [ ] I think you missed some of the requirements of the Books component. It should display a list of books and should have a form for adding a book.

> Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet)
Should have a form for adding a book (no functionality yet)

Reply:
Ok sure. It actually skipped my mind. Let me work that on too. 👍